chemie/events/views.py

```diff
@@ -36,12 +36,14 @@ def create_event(request):
 
 def list_all(request):
     all_events = Event.objects.filter(date__gt=timezone.now()).order_by('date')
+    past_events = Event.objects.filter(date__lte=timezone.now()).order_by('date')
     my_events = None
     if request.user:
         my_events = Event.objects.filter(attendees__username__exact=request.user)
     context = {
         'events': all_events,
         'my_events': my_events,
+        'past_events': past_events,
```

Comment:
Nokke e ikke tenkte på her e at d blir mange ting etter hvert.. Kanskje vi må lage egne views for "my events" og "past events"?
Akkurat no henta den egentli alle verdens arrangement, sjølv om du bare e intressert i dem fremtidige feks